Week3/homework/step2-3.js

```diff
@@ -31,8 +41,15 @@ function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  if (num <= 0) {
+    return result;
+  }
```

Comment:
Here you are absolutely correct to test the value of `num` before entering the `do-while` loop as now the loop condition is tested at the end of the loop.